package.json

```diff
@@ -91,9 +92,9 @@
     "react-devtools-core": "3.6.0",
     "react-dom": "16.7.0",
     "react-test-renderer": "16.8.3",
-    "reactotron-react-native": "^2.1.0",
-    "reactotron-redux": "^2.1.0",
-    "reactotron-redux-saga": "^3.0.0",
+    "reactotron-react-native": "^3.6.4",
```

Comment:
The API you were using for `reactotron-redux` was the 3.x.x version but you had the 2.x.x version installed. Just upgraded everything for good measure.